query-node/mappings/src/workingGroups.ts

```diff
@@ -744,6 +752,20 @@ export async function workingGroups_WorkerRemarked({ store, event }: EventContex
     const actor = await getWorker(store, group.name, workerId)
 
     await moderatePost(store, event, 'workerRemark', postId, actor, rationale)
+  } else if (metadata?.verifyValidator) {
+    if (group.name !== 'operationsWorkingGroupBeta') {
+      return invalidMetadata(`The ${group.name} cannot verify validator accounts`)
+    }
+
+    const actor = await getWorker(store, group.name, workerId)
+    if (actor === undefined) {
+      return invalidMetadata(`${actor} is not an HR account`)
+    }
+
+    const member = await getMemberById(store, workerId, ['metadata'])
```

Comment:
workerId is the memberId of the HR worker not that of the validator.

`const {memberId, validatorAccount} =  const { postId, rationale } = metadata.verifyValidator`

The memberId is string but I need u64 for the getMemberById function. What should I do?

Reply:
Oh I completely missed that.

You could use:
```ts
import { createType } from '@joystream/types'
createType('MemberId', Number(metadata.verifyValidator.memberId))
```
But `getMemberById` just converts the id into a string so maybe just change the `getMemberById` `id` argument to: `MemberId | string`.